### DISCUSSION TITLE
Multi openAPI documents on same route page

### DISCUSSION THREAD

**Discussion by WeihanLi** [scalar/scalar#3909] [Q&A] [answered]
Does scalar support show multiple open API documents on the same route page like swagger ui?

![image](https://github.com/user-attachments/assets/7085c2e2-4807-4045-a548-96c557b099b2)

**Comment by xC0dex** [comment, marked as answer]
Hey @WeihanLi, this feature is requested [here](https://github.com/scalar/scalar/issues/1851)!

**Reply by WeihanLi** [reply]
Great to know, thanks very much for the info
